honeybadger.go

```diff
@@ -0,0 +1,83 @@
+package cleisthenes
+
+type HoneyBadger struct {
+
+	// TODO : HoneyBadger must have ACS
+
+	// TODO : HoneyBadger must have consensused batch
+
+	// que have transaction FIFO queue.
+	que queue
+
+	// current epoch
+	epoch int
+}
+
+// Add transaction to FIFO queue
+func (hb *HoneyBadger) AddTransaction() {
+	panic("implement me w/ test case :-)")
+}
+
+// Send batch to ACS
+func (hb *HoneyBadger) SendBatch() {
+	panic("implement me w/ test case :-)")
+}
+
+// Callback after receive message from other nodes
+func (hb *HoneyBadger) HandleMessage() error {
+	panic("implement me w/ test case :-)")
+}
+
+// Define transaction interface.
+type Transaction interface {}
+
+// Define transaction FIFO queue
+type queue struct {
```

Comment:
how about add `type queue interface` and change current `queue` into `memQueue` ?

Reply:
Why do you think about that?